docs/DXIL.rst

```diff
@@ -515,33 +515,52 @@ Additional shader properties are specified via tag-value pair list, which is the
+5        Low-precision data types                                                                 Use of half or i16 data types
+6        Double-precision extensions                                                              Use of FDiv, UIToFP, SIToFP, FPToUI, FPToSI, or Fma instructions with a double type
+7        Masked sum of absolute differences                                                       Use of the Msad instruction
+8        All resources bound for the duration of shader execution                                 Command-line flag ``/all_resources_bound`` provided to DXC
+9        View port or render target array index from any shader feeding rasterizer                ViewPortArrayIndex or RenderTargetArrayIndex semantics are present in the input signature of a geometry shader, or the output signature of a vertex, domain, or hull shader. If DXIL validator version < 1.4, this flag is also required if the ViewPortArrayIndex or RenderTargetArrayIndex semantics are present in the input signature of domain and hull shaders
+10       Inner coverage                                                                           Use of the InnerCoverage instruction, or the InnerCoverage semantic is present in the output signature of a pixel shader
+11       Stencil reference value                                                                  The StencilRef semantic is present in the output signature of a pixel shader
+12       Tiled resources                                                                          Use of the CheckAccessFullyMapped instruction or, if DXIL validator version >= 1.8, the use of LodClamp in intrinsics SampleGrad, SampleCmpGrad, Sample, SampleBias, SampleCmp, or SampleCmpBias
+13       Typed UAV load additional formats                                                        Use of TextureLoad or BufferLoad on a UAV with a multi-component data type. If DXIL validator version == 1.0 then this flag is set when a TextureLoad or BufferLoad is used on any UAV, regardless of its data type
+14       Comparison filtering for feature level 9                                                 Not set
+15       64 UAV slots                                                                             The presence of more than 8 UAVs. If Shader Model >= 6.6, then UAV ranges count as multiple UAVs
+16       UAVs at every shader stage                                                               The shader has UAVs and: if Shader Model >= 6.8, the shader is a vertex, hull, domain, or geometry shader; else (Shader Model < 6.8) the shader is not a compute or pixel shader
+17       CS4 raw and structured buffers                                                           Use of RawBuffer or StructuedBuffer in a Shader Model 4.x compute shader
+18       Rasterizer order views                                                                   Not set
```

Comment:
Is this really not set in DXC?
@tex3d, do we not set this flag on ROV usage?

Reply:
Oh yea it is. I just missed it. It's set in [DxilModule.cpp:L315](https://github.com/microsoft/DirectXShaderCompiler/blob/2b1c2e640dae09adf1cb2dd52bc5ce860d73b02b/lib/DXIL/DxilModule.cpp#L315). I will update the flag criteria